src/prefect/executors/dask.py

```diff
@@ -512,11 +511,11 @@ def _interrupt_pool(self) -> None:
         if self._pool is None:
             return
 
-        # Terminate the pool
-        self._pool.terminate()
+        # Shutdown the pool
+        self._pool.shutdown(wait=False)
```

Comment:
Let's cancel any unstarted futures as well
```suggestion
        self._pool.shutdown(wait=False, cancel_futures=True)
```

Reply:
@jacques- my apologies, looks likes this is a 3.9+ only feature. I'm okay with excluding it entirely instead of toggling by version.